4001-5000/LIT4145Onausp.xml

```diff
@@ -58,7 +67,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <ab>ሐሳበ፡ ጽንዕት፡ ወጽንጽንት። በመስከረም። ፭፡ ጽንዕት። ወእምዝ፡ ጽንጽንት፡ በጥቅምት፡ ፳ወ፫። ጽንዕት። ወእምዝ፡ ጽንጽንት። </ab>
             </div>
          </div>
-       
+       <div type="bibliography">
+          <listBibl type="secondary">
+             <bibl><ptr target="bm:Grebaut1913Joursfastes"/></bibl>
```

Comment:
You can add A. Gasparini, "Di due vocabili etiopici oscuri" (RSE 1946 accessible online); I wonder if his interpretation works out, but he lists more mss, you can check them and harmonize.

Reply:
All manuscripts checked and added to the list. Except for BnF d'Abbadie 186, which does not exist as a stub, yet. Do you want me to create a stub for it and add it to the list?

Reply:
I will do this.

Reply:
@DenisNosnitsin1970 should we wait?

Reply:
@CarstenHoffmannMarburg I do not know when @DenisNosnitsin1970 will get to it, but you can still simply add BNFabb186 link, even if it for the moment would lead into nowhere.